rules-xml/api/src/main/resources/CatalogManager.properties

```diff
@@ -0,0 +1,8 @@
+#CatalogManager.properties
+verbosity=1
+relative-catalogs=false
+# Always use semicolons in this list
+catalogs=./core-catalog.xml
+prefer=public
+static-catalog=yes
+allow-oasis-xml-catalog-pi=yes
```

Comment:
I added only W3C html/xhtml dtds/schemas. If we would like to optimize more validations like JBoss project/product dtds/schemas or purely Java EE ones, we can add later more definition in core-catalog.xml or add more catalogs for every possible thing.